.github/workflows/test-server-all.yml

```diff
@@ -50,14 +50,38 @@ jobs:
         env:
           BAZEL_CACHE_BUCKET: ${{ secrets.BAZEL_CACHE_BUCKET }}
 
-      - name: Build images and run computation_container
-        run: make test t=./computation_container m=build p=small
-        working-directory: ./scripts
+      - name: Cache Docker layers
+        uses: actions/cache@v3
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-small_cc-${{ hashFiles('./packages/server/computation_container/**') }}
+          restore-keys: ${{ runner.os }}-small_cc-
```

Comment:
@sakamoto-souta 
I feel uncomfortable with the suffix ending in -.
Is there an intention behind this naming?

Reply:
I just imitated the [official example](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#example-using-multiple-restore-keys).
I don't have any intention.